src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -186,6 +206,19 @@ public FormValidation doCheckReturnStatus(@QueryParameter boolean returnStdout,
                     LOGGER.log(Level.FINE, "Jenkins is not running, no such node {0}, or it is offline", node);
                     return null;
                 }
+                if (watching) {
+                    try {
+                        controller.watch(ws, new HandlerImpl(this, ws, listener()), listener());
+                        recurrencePeriod = WATCHING_RECURRENCE_PERIOD;
+                    } catch (UnsupportedOperationException x) {
+                        getContext().onFailure(x);
+                    } catch (Exception x) { // as below
+                        LOGGER.log(Level.FINE, node + " is evidently offline now", x);
```

Comment:
Apparently it's not documented in https://github.com/jenkinsci/durable-task-plugin/pull/60

Reply:
Execution link in the log would be helpful

Reply:
(for #63)